x/stakeibc/client/cli/query_test.go

```diff
@@ -0,0 +1,45 @@
+package cli_test
+
+import (
+	"testing"
+)
+
+func TestQueryParams(t *testing.T) {
```

Comment:
we can probably just remove this file if all these are empty

Reply:
Done cfd6422